### DISCUSSION TITLE
Are any collaborators will be lost when I downgrade from GitHub enterprise for organizations to GitHub free or Teams?

### DISCUSSION THREAD

**Discussion by Ahmedelsa3eed** [community/community#28568] [Enterprise] [answered]
In general, What features will be lost while downgrading from GitHub enterprise?

**Comment by pasindu651** [comment, marked as answer]
In general, you will lose access to compliance, advanced security, deployment controls, and the following (if you downgrade to GitHub FREE):
- Required reviewers in private repositories
- Protected branches in private repositories
- Code owners in private repositories
- GitHub Pages in private repositories
- Wikis in private repositories
- Multiple assignees for issues and pull requests in private repositories
- Multiple pull request reviewers in private repositories
- Draft pull requests in private repositories
- Autolinked references in private repositories
- Standard support
[Learn more here](https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/faq-about-changes-to-githubs-plans)
To answer your question, access to advanced collaboration will be lost if you downgrade. Hope this helps!

**Reply by Ahmedelsa3eed** [reply]
@pasindu651 Thanks a lot for your time

**Reply by Ahmedelsa3eed** [reply]
Do you know what **advanced collaboration** mean?
